### DISCUSSION TITLE
Npm run dev command is not working.

### DISCUSSION THREAD

**Discussion by rizwanshah1098** [tailwindlabs/tailwindcss#14047] [Help]
npm error Missing script: "dev"
npm error
npm error To see a list of scripts, run:
npm error   npm run
npm error A complete log of this run can be found in: C:\Users\rizwa\AppData\Local\npm-cache\_logs\2024-07-24T09_41_16_871Z-debug-0.log

**Comment by wongjn** [comment]
This doesn't seem related to Tailwind. It seems like you are trying to run the `dev` npm script command, but where the project does not have it defined. Consider checking [the npm documentation on scripts](https://docs.npmjs.com/cli/v10/using-npm/scripts).
